cmd/csaf_aggregator/indices.go

```diff
@@ -311,6 +351,7 @@ func (w *worker) writeService() error {
 func (w *worker) writeIndices() error {
 
 	if len(w.summaries) == 0 || w.dir == "" {
+		w.writeROLIENoSummaries("empty")
```

Comment:
Hmm. `empty` sounds interesting... Maybe, we should use `undefined`? 
Thoughts?

Reply:
Using undefined now, am open to change if necessary.  (https://github.com/csaf-poc/csaf_distribution/pull/357/commits/2a833bec790477e86c36cd90f43bf260c43c0c3e)

Also: Provider was already able to generate the ROLIE feeds, but the default was set not to. Now the default is to set to generate the feed. (https://github.com/csaf-poc/csaf_distribution/pull/357/commits/418d93bde0ca1a364656a3cee7fcb6f75690c48e)